app/src/org/commcare/tasks/UpdateTask.java

```diff
@@ -108,8 +108,7 @@ public void startPinnedNotification(Context ctx) {
         } catch (InvalidResourceStructureException e) {
             ResourceInstallUtils.logInstallError(e,
                     "Structure error ocurred during install|");
-            String structureError = "CommCare found an issue with the '" + e.resourceName + "' resource:\n" + e.getMessage();
-            return new ResultAndError<>(AppInstallStatus.UnknownFailure, structureError);
+            return new ResultAndError<>(AppInstallStatus.UnknownFailure, e.resourceName + "==" + e.getMessage());
```

Comment:
Ahh, gotcha. Would you mind adding a comment to that effect? Good to pull after!